src/sorunlib/stimulator.py

```diff
@@ -198,3 +198,86 @@ def calibrate_gain(duration=60, speed_rpm=90,
 
         if stop:
             _stop()
+
+
+def calibrate_gain_tau(duration_gain=60, duration_tau=10, duration_stabilization=10,
+                       speed_rpm_gain=90, speeds_rpm_tau=[225, 495, 945, 1395, 1845, 2205],
+                       forward=True, do_setup=True,
```

Comment:
`do_setup` is missing in the docstring.

EDIT: Ah, it looks like 'setup' is always done, so this should just be removed.